xapian-core/cluster/cluster.cc

```diff
@@ -1,8 +1,8 @@
 /** @file cluster.cc
- *  @brief Xapian::Cluster API class
+ *  @brief Cluster API
  */
-/* Copyright (C) 2016 Richhiey Thomas
- * Copyright (C) 2016 Richard Boulton
+/* Copyright (C) 2010 Richard Boulton
+ * Copyright (C) 2016 Richhiey Thomas
```

Comment:
This would ideally have been two commits: one to fixup the previous work, and one (separately) to implement `RoundRobin`. Each commit should do one, complete, thing (or, in the case of a fixup, should combine with a previous commit to do one, complete, thing).